gr-analog/python/analog/wfm_rcv_pll.py

```diff
@@ -61,15 +60,14 @@ def __init__(self, demod_rate, audio_decimation):
         self.fft_filter_xxx_1.declare_sample_delay(0)
         self.blocks_multiply_xx_2 = blocks.multiply_vff(1)
         self.blocks_multiply_xx_0 = blocks.multiply_vcc(1)
-        self.blocks_multiply_const_vxx_0_1 = blocks.multiply_const_ff(5.5)
-        self.blocks_multiply_const_vxx_0 = blocks.multiply_const_ff(-5.5)
         self.blocks_complex_to_imag_0 = blocks.complex_to_imag(1)
-        self.blocks_add_xx_0_0 = blocks.add_vff(1)
+        self.blocks_sub_xx_0 = blocks.sub_ff(1)
         self.blocks_add_xx_0 = blocks.add_vff(1)
         self.analog_quadrature_demod_cf_0 = analog.quadrature_demod_cf(demod_rate/(2*math.pi*deviation))
         self.analog_pll_refout_cc_0 = analog.pll_refout_cc(0.001, 2*math.pi * 19200 / demod_rate, 2*math.pi * 18800 / demod_rate)
-        self.analog_fm_deemph_0_0 = analog.fm_deemph(fs=48000, tau=75e-6)
-        self.analog_fm_deemph_0 = analog.fm_deemph(fs=48000, tau=75e-6)
+        self.analog_fm_deemph_0_0 = analog.fm_deemph(fs=demod_rate//audio_decim, tau=75e-6)
```

Comment:
fs is a float so no need to convert to integer. It should be whatever comes out of demo_rate/audio_decim so that the fm_deemph filter response is correct. Also maybe create an audio_rate variable right after demod_rate instead of doing the divides here in the argument list.

Reply:
I've added the audio_rate variable, and also added deemph_tau as a parameter, since the de-emphasis time constant will be different between US and EUR locales.